applications/dashboard/src/scripts/forms/DashboardRadioGroups.tsx

```diff
@@ -46,7 +47,7 @@ export function DashboardRadioGroup(props: IProps) {
     return (
         <RadioGroupContext.Provider value={props}>
             <div
-                className={classNames(rootClass, { inline: props.isInline })}
+                className={classNames(rootClass, { inline: props.isInline }, { grid: props.isGrid })}
```

Comment:
This is not incorrect, but FYI, you could of combined the two conditional class objects into 1 here.